### PR TITLE
[Example] TAHIN

### PR DESCRIPTION
**Pull request by KounianhuaDu**
## Description
This is the implementation of the TAHIN module proposed in "A Heterogeneous Information Network based Cross DomainInsurance Recommendation System for Cold Start Users".

## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the my best knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

**Comment by KounianhuaDu** [comment]
@zhjwy9343 
In addition, should I send an email to the author with the pr link attached?

File: examples/pytorch/TAHIN/readme.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# DGL Implementation of the TAHIN
+
+This DGL example implements the TAHIN module proposed in the paper [HCDIR](https://arxiv.org/pdf/2007.15293.pdf). Since the code and dataset have not been published yet, we implement its main idea and emperiment on two other datasets.
```
Comment (zhjwy9343) [inline examples/pytorch/TAHIN/readme.md:3]: emperiment -> experiment

**Review by zhjwy9343 — CHANGES_REQUESTED** [review]

**Review by zhjwy9343 — APPROVED** [review]
LGTM

**Review by zhjwy9343 — APPROVED** [review]
LGTM

**Pull request merged**